.travis.yml

```diff
@@ -14,13 +14,23 @@ before_script:
   - docker info
   - docker version
 script:
+  # Enable multi-architecture containers using QEMU
   - docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
   - docker buildx create --name xbuilder --use
+  # Login to Docker Hub and configure push behaviour if we are on master
   - >
     if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
       echo "$DOCKER_PASS" | docker login -u ${DOCKER_USER} --password-stdin
       dopush="--push"
     fi
-  - docker buildx build --progress plain --platform=linux/amd64,linux/arm64,linux/arm/v7 ${dopush} -t ${DOCKER_USER}/docker-magicmirror:latest .
+  # Build the latest stable MagicMirror release (https://stackoverflow.com/a/51761312/4934537)
+  - >
+    latest_release=$(git ls-remote --tags --refs --sort="v:refname" https://github.com/MichMich/MagicMirror.git | tail -n1 | sed 's/.*\///')
```

Comment:
I think there is a `\n` in `latest_release` which causes the problem (had something similar yesterday which costs me 3 hours ...), try e.g.

````
git ls-remote --tags --refs --sort="v:refname" https://github.com/MichMich/MagicMirror.git | tail -n1 | sed 's/.*\///' | tr '\n' ' '
````

Reply:
Oh please let this be it! I'm already investigating over an hour...
I'll try this immediately!

Reply:
This didn’t fix the issue 😭

Reply:
no :(

may like this. If it does'nt work we may get a better error message ...

````
  - latest_release=$(git ls-remote --tags --refs --sort="v:refname" https://github.com/MichMich/MagicMirror.git | tail -n1 | sed 's/.*\///' | tr '\n' ' ')
  - >
    if [ "$(docker manifest inspect bastilimbach/docker-magicmirror:"${latest_release}" > /dev/null; echo $?)" != 0 ]; then
      docker buildx build --progress plain --platform=linux/amd64,linux/arm64,linux/arm/v7 "${dopush}" --build-arg branch="${latest_release}" -t "${DOCKER_USER}"/docker-magicmirror:"${latest_release}" -t "${DOCKER_USER}"/docker-magicmirror:latest .
    fi
````

Reply:
Ok lets try it... If this doesn't print any useful error message I will try a separate bash file.

Reply:
Ok we really got a better error message:
```
0.42s$ latest_release=$(git ls-remote --tags --refs --sort="v:refname" https://github.com/MichMich/MagicMirror.git | tail -n1 | sed 's/.*\///' | tr '\n' ' ')
The command "latest_release=$(git ls-remote --tags --refs --sort="v:refname" https://github.com/MichMich/MagicMirror.git | tail -n1 | sed 's/.*\///' | tr '\n' ' ')" exited with 0.
0.10s$ if [ "$(docker manifest inspect bastilimbach/docker-magicmirror:"${latest_release}" > /dev/null; echo $?)" != 0 ]; then
  docker buildx build --progress plain --platform=linux/amd64,linux/arm64,linux/arm/v7 "${dopush}" --build-arg branch="${latest_release}" -t "${DOCKER_USER}"/docker-magicmirror:"${latest_release}" -t "${DOCKER_USER}"/docker-magicmirror:latest .
fi
invalid reference format
"docker buildx build" requires exactly 1 argument.
See 'docker buildx build --help'.
Usage:  docker buildx build [OPTIONS] PATH | URL | -
Start a build
The command "if [ "$(docker manifest inspect bastilimbach/docker-magicmirror:"${latest_release}" > /dev/null; echo $?)" != 0 ]; then
  docker buildx build --progress plain --platform=linux/amd64,linux/arm64,linux/arm/v7 "${dopush}" --build-arg branch="${latest_release}" -t "${DOCKER_USER}"/docker-magicmirror:"${latest_release}" -t "${DOCKER_USER}"/docker-magicmirror:latest .
fi
" exited with 1.
```

Reply:
Probably because "latest_release" is not available in the next command.